### DISCUSSION TITLE
Set charge current

### DISCUSSION THREAD

**Discussion by bperemans** [nathanmarlor/foxess_em#194] [Q&A]
Is it possible to set a specific charge current with a force charge? In Belgium we have capacity fee (we pay more for peaks). It would be great to set charge current for force charge low, but still be able to charge with solar at high current.

**Comment by nathanmarlor** [comment]
Good idea - this could potentially be included within the start force charge service call?

**Reply by screenagerbe** [reply]
Also a Belgian user here.
I'm still on the old analogue meter, so it does not matter for me atm, but the meter will be replaced by a digital meter in the coming weeks. I've been playing with the time periods to stay below a certain threshold (peak average of 3.5 kW/15minutes). But it would be easier if the integration could keep that threshold in mind when charging from the grid.

The main problem is that this so called capacity tariff is based on the yearly average of the average month peak, which is based on your peak consumption over 15 minutes (0-14,15-29,30-44,45-59). It also takes your yearly total consumption in consideration (the higher your consumption, the higher your allowed peak). So this average peak value will be different for each user in the Flemish region of Belgium, captar is not in use in Brussels or Wallonia.
The minimum captar is set at 2.5 kW/15 minutes (but this might change off course due to legislation).

Not really sure how I would like this to be. I guess I'd like a sensor which controls the charge current but isn't linked to the service call at all. It would give me the possibility to create multiple charge windows (e.g. Time Periods) for charging the batteries. 

p.s. 
I'm also using the foxess-modbus integration
